judges/quality-of-service/quality-of-service.rb

```diff
@@ -73,11 +73,13 @@
   # Average issues
   issues = []
   Fbe.unmask_repos.each do |repo|
-    Fbe.octo.repository(repo).then do |json|
-      issues << json[:open_issues]
+    (f.since.utc.to_date..Time.now.utc.to_date).each do |date|
+      issues << Fbe.octo.search_issues(
+        "repo:#{repo} type:issue created:#{date.iso8601[0..9]}..#{date.iso8601[0..9]}"
+      )[:total_count]
```

Comment:
@Yegorov I'm not sure this is the right logic. You count the number of opened issues. However, if all of them were opened and immediately closed, the backlog size is zero, while your counter will give big numbers. WDYT?

Reply:
> However, if all of them were opened and immediately closed, the backlog size is zero

@yegor256 I search only created issues by day, it is not check open state.  
This query:
```ruby
"repo:#{repo} type:issue created:#{date.iso8601[0..9]}..#{date.iso8601[0..9]}"
```
not has [`state:open`](https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-by-open-or-closed-state) and return all issues created by specified day

Reply:
@Yegorov let's say I have three issues created on 2024-05-06 and all of them were closed the same day. What will be the size of the backlog at that day?

Reply:
@yegor256 three issues.

Reply:
@Yegorov what if I create 3 issues on 2024-05-06 and then 4 issues on 2024-05-07, and then close them all on 2024-05-08 -- what will be backlog sizes on these three days?

Reply:
@yegor256 
|date|count|
|------------|---|
| 2024-05-06 | 3 |
| 2024-05-07 | 4 |
| 2024-05-08 | 0 |

`average_backlog_size = (3 + 4 + 0)/3 = 2.33`

Reply:
@Yegorov exactly. However, the right answer is this:

|date|count|
|------------|---|
| 2024-05-06 | 3 |
| 2024-05-07 | 7 |
| 2024-05-08 | 0 |

`average_backlog_size = (3 + 7 + 0)/3 = 3.33`

Reply:
@yegor256 fixed, please check